src/main/scala/io/iohk/ethereum/blockchain/sync/regular/BlockFetcher.scala

```diff
@@ -44,7 +44,7 @@ class BlockFetcher(
   import BlockFetcher._
 
   implicit val ec: ExecutionContext = context.dispatcher
-  implicit val timeout: Timeout = syncConfig.peerResponseTimeout + 1.second // some margin for actor communication
+  implicit val timeout: Timeout = syncConfig.peerResponseTimeout + 2.second // some margin for actor communication
```

Comment:
Add value if we put this time margin as config?

Reply:
Hmm would we ever want to change this again? Maybe we could have a general mantis configuration of akka ask timeout margins (as we are using this 2 seconds in other places)

Either way I'd do that on a separate task

Reply:
That sound good